src/api/registerForEvents.js

```diff
@@ -11,7 +11,6 @@ export default (auth: Auth) =>
         'update_message',
         'subscription',
         'reaction',
-        'presence',
```

Comment:
That is not how the call works. When you pass the object **none** are included in the initial call. These specify what events to get later.